auth/test_jwt.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from auth import jwt
+from contextlib import contextmanager
+
+
+@contextmanager
+def does_not_raise():
+    yield

```

Comment:
FYI, you do not need this anymore. https://docs.pytest.org/en/latest/example/parametrize.html#parametrizing-conditional-raising